mk/config.mk

```diff
@@ -104,5 +104,5 @@ TA_SIGN_KEY ?= keys/default_ta.pem
 # may not because they obtain the isoc functions from elsewhere
 CFG_LIBUTILS_WITH_ISOC ?= y
 
-# Floating point support for TAs default on only for arm32 core
-CFG_TA_FLOAT_SUPPORT ?= $(CFG_ARM32_core)
+# Enables floating point support for user TAs
```

Comment:
Add a comment such as "This is hard float support unless CFG_WITH_VFP=n and TEE core is 32-bits" (I hope I understand correctly)